manifests/gcp_marketplace/schema.yaml

```diff
@@ -108,7 +108,17 @@ properties:
     type: string
     x-google-marketplace:
       type: NAMESPACE
-
+  GCSbucketName:
```

Comment:
I saw other property name‘s first char is lowercase. how about we follow the practice?

Reply:
Feel free to ask Yang to give a LGTM to submit if I'm not online.

Reply:
Per discussion now we're moving towards full automated bucket provisioning. Thus this will be redundant.